src/components/CreateRules/UploadRules.vue

```diff
@@ -73,4 +227,36 @@ export default class UploadRules extends Vue {
 .v-icon.mdi-information-outline {
   margin-top: -2px;
 }
+
+.help-btn {
+  cursor: pointer;
+  color: $app-blue;
+  vertical-align: middle;
+}
+
+.create-rules-help {
+  margin: 2rem 0;
```

Comment:
Ditto for help-section-label-top and help-section-label: it's sometimes/usually easier to use the Vuetify spacing helpers (eg, mt-4) for layout purposes, in the template, instead of using a classname that a reviewer has to look up to see what the layout is.
Again, YMMV. This is not a blocking change for this PR.